### PR TITLE
Rest api and jwt security

### PR DESCRIPTION
**Pull request by dreamlandliu**
## Modules
- [ ] UI
- [x] Controller
- [ ] Agent
- [ ] Client
- [ ] Python-SDK
- [ ] Others

## Checklist
- [ ] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Comment by tianweidut** [comment]
@dreamlandliu  `git rebase upstream main` or `git rebase origin main` to rebase some merge commit?
image

File: server/controller/src/main/java/ai/starwhale/mlops/api/protocol/ResponseMessage.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+public class ResponseMessage<T> {
+    private static final String requestId = null;
+
+    /**
+     * 默认的成功码
```
[the remark this answers is not in the document]
Comment (dreamlandliu) [reply server/controller/src/main/java/ai/starwhale/mlops/api/protocol/ResponseMessage.java]: removed this class

File: server/controller/src/main/java/ai/starwhale/mlops/api/protocol/ResponseMessage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        setMessage(message);
+    }
+
+    public ResponseMessage(String code, String message) {
+        setCode(code);
+        setMessage(message);
+    }
+
+    public static <T> ResponseMessage<T> asSuccess(T data) {
+        // todo 待确定如何国际化
```
[the remark this answers is not in the document]
Comment (dreamlandliu) [reply server/controller/src/main/java/ai/starwhale/mlops/api/protocol/ResponseMessage.java]: removed this class

**Comment by goldenxinxing** [comment]
> @dreamlandliu `git rebase upstream main` or `git rebase origin main` to rebase some merge commit? image

Checking out from master every time at the beginning could avoids this commit

**Review by tianweidut — APPROVED** [review]
LGTM

**Pull request merged**